apd/move_urls_from_descriptions.rb

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/ruby -w
+
+require 'json'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.